src/csharp/Grpc.Core.Api/IClientStreamWriter.cs

```diff
@@ -34,5 +35,16 @@ public interface IClientStreamWriter<in T> : IAsyncStreamWriter<T>
         /// Completes/closes the stream. Can only be called once there is no pending write. No writes should follow calling this.
         /// </summary>
         Task CompleteAsync();
+
+#if NETSTANDARD2_1_OR_GREATER
+        /// <summary>
+        /// Completes/closes the stream. Can only be called once there is no pending write. No writes should follow calling this.
+        /// </summary>
+        /// <param name="cancellationToken">Cancellation token that can be used to cancel the operation.</param>
+        Task CompleteAsync(CancellationToken cancellationToken)
```

Comment:
Also this overload only exists for client-side call (since this is IClientStreamWriter) and on writer we already have a simple mechanism for cancelling a call (you pass a cancellation token to a newly created call), so adding `CompleteAsync(CancellationToken cancellationToken)` overload is not going to help with the situation on the server and it's redundant on the client.

Reply:
Makes sense. I'll remove it.